tools/sort.py

```diff
@@ -302,6 +302,7 @@ def sort_face_dissim(self):
         return img_list
 
     def sort_face_cnn(self):
+        import_face_recognition()
```

Comment:
I can't see that face_recognition is used in this method. Just FaceLandmarksExtractor